cookbooks/web/recipes/base.rb

```diff
@@ -17,13 +17,6 @@
 # limitations under the License.
 #
 
-node.set[:nfs]["/store/rails"] = {
-  :host => node[:web][:fileserver],
-  :path => "/store/rails"
-}
-
-include_recipe "nfs"
-
```

Comment:
I was testing the cookbook locally, and it was difficult given that I don't have an NFS server locally to point it to. So I moved that bit out of `base` into `nfs` and included that from the `rails` and `gpx` recipes where it's actually used.